C/services/south/ingest.cpp

```diff
@@ -28,6 +28,157 @@ static void ingestThread(Ingest *ingest)
 	}
 }
 
+/**
+ * Create a row for given assetName in statistics DB table, if not present already
+ * The key checked/created in the table is "SOUTH_READINGS_FROM_<assetName>"
+ * 
+ * @param assetName     Asset name for the plugin that is sending readings
+ */
+int Ingest::CreateStatsDbEntry(const string& assetName)
+{
+	// Prepare foglamp.statistics update
+	string statistics_key = "south_readings_from_" + assetName;
```

Comment:
Could we make this a bit shorter, maybe just the asset name to INGEST_<assetName> - the risk is with a long asset name we end up overflowing the database column.

Reply:
Ok sure.